null_safety_examples/null_safety_codelab/bin/more_nullable_types.dart

```diff
@@ -0,0 +1,9 @@
+void main() {
+  List<String> aListofStrings = ['one', 'two', 'three'];
+  List<String> aNullableListOfStrings;
```

Comment:
when i change the aNullableListOfStrings and aListofNullableStrings to String? as instructed, i still see this error in Dartpad:

```
The non-nullable local variable 'aNullableListOfStrings' must be assigned before it can be used - line 12
```

Reply:
so, maybe just this?

  List<String?> aNullableListOfStrings = [];

Reply:
lgtm